marketplace_builder/views/pages/how-to/integrations/integrating-slack-api-call-notifications.liquid

```diff
@@ -0,0 +1,90 @@
+---
+converter: markdown
+metadata:
+  title: Integrating Slack Using API Call Notifications 
+slug: how-to/integrations/integrating-slack-api-call-notifications
+---
+
+This guide will help you create a Slack integration for Platform OS using API Call Notifications. 
+
+## Requirements
+To follow this tutorial, you should be familiar with creating API call notifications. You also have to have access to a Slack workspace, have enabled webhooks, and created an incoming webhook. You will need the webhook URL for creating the API call notification.  
+
+* [Creating an API Call Notification](/get-started/notifications/creating-api-call-notification)
+* [Creating an Incoming Webhook (Slack Documentation)](https://api.slack.com/incoming-webhooks#getting-started)
+
+## Steps
+
+Integrating Slack using API call notifications is a two-step process:
+
+1. Create API call notification
+2. Add API call notification to form
+
+### Step 1: Create API call notification
+
+Your webhook URL looks similar to this: `https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX`.
+Create an API call notification using your own webhook URL (replace placeholder):
+
+{% raw %}
+
+<pre><code class="language-liquid">
+---
+name: { name of your file here }
+delay: 0
+enabled: true
+format: http
+headers: '{ "Content-Type": "application/json" }'
+request_type: POST
+to: { https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX }
+trigger_condition: t
+---
+{
+	"text": "New Contact on Platform.OS.com!\nName: {{ form.properties.first_name }} {{ form.properties.last_name }}\nEmail: {{ form.properties.email }}\nPhone: {{ form.properties.phone }}\nComments: {{ form.properties.description }}"
```

Comment:
those variables depends on a form, and we used `{ your something here }`, so I think it would be enough to just keep`"text": "New Contact on Platform.OS.com! `here, without variables which will in 99.9999999% cases will not work in a random form. Alternatively, we could do something like:
`"text": "New Contact on Platform.OS.com: {{ form.email }} !`